### DISCUSSION TITLE
macOS Big Sur can't be detected?

### DISCUSSION THREAD

**Discussion by kleuter** [plausible/analytics#1263] [Anything Else]
Hi,

I just checked both GA and plausible - both do not detect macOS Big Sur (10.16 or 11.0). Same day stats:

![sshot-6](https://user-images.githubusercontent.com/2600624/130366529-f9132b31-0b28-4d7e-a899-dfd6c6197938.png)
![sshot-7](https://user-images.githubusercontent.com/2600624/130366531-2241855b-3e2f-4a63-bf93-eafbf188861c.png)

How is that? Thanks.
Surely believe me there are plenty of website users with with Big Sur.

**Comment by ukutaht** [comment]
That's strange indeed.

Looks like Safari on Mac intentionally hides the Big Sur version number: https://bugs.webkit.org/show_bug.cgi?id=216593 I'm not sure what Chrome/Firefox are doing on Big Sur.

**Comment by kleuter** [comment]
Thanks for the follow up, according to the thread Chrome and Firefox decided to cap the User-Agent string's reported macOS version at 10.15 as Safari does.

**Reply by ukutaht** [reply]
That sucks. I don't see much we could do on our end to detect it then
